CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+[Version 0.4.1](https://github.com/novoda/gradle-build-properties-plugin/releases/tag/v0.4.1)
+--------------------------
+
+- Remove `@Deprecated` annotation for BuildProperties.file build properties through project properties ([#PR59](https://github.com/novoda/gradle-build-properties-plugin/pull/59)).
```

Comment:
`BuildProperties.file`?

Reply:
`BuildProperties#file` method? how do you specify method/function within a class

Reply:
Also it's quite confusing to mention project in there. What about:
```
Remove `@Deprecated` annotation for `BuildProperties.file()` as the IDE highlights wrong `Project.file()` method as deprecated.
```

Reply:
but wait this sentence doesn't make sense anyway :thinking:

Reply:
commit 48e5d2bb81eb61f2750e5014bcafed36debbd40a
Author: Ataul Munim <hello@ataulm.com>
Date:   Tue Apr 17 15:31:52 2018 +0100

    Update changelog to highlight why we're removing deprecation

Reply:
" build properties through project properties " was the remaining copy&paste from the previous changelog :joy: